### PR TITLE
translate plugins: dashboard ( partial ) for pt_BR language package

### PR DESCRIPTION
**Pull request by marciojv**

### CODE REVIEW

File: data/pt_BR/system/dashboards/resources/gwt/schedulingdialogs/messages/mantleMessages_pt_BR.properties [hunk 1]
```diff
@@ -512,22 +512,22 @@ removePermissions=Remove all permissions<TRANSLATE ME>
...
+short=Curto
+verbose=Verboso
+scheduling=Agendamento
+schedulable=Permitir Agendamento
+hidden=Ocultar
+owner=Proprietário
+fromName=Nome De
+permissionsWillBeLostQuestion=Herdar permissões de pasta irá remover quaisquer permissões selecionadas.
+permissionsWillBeLostConfirmMessage=Herdar Permissões de Pasta
+couldNotGetPermissions=Não é possível herdar permissões de pasta {0}<TRANSLATE ME>
```
Comment (carlosrusso) [inline data/pt_BR/system/dashboards/resources/gwt/schedulingdialogs/messages/mantleMessages_pt_BR.properties]: you forgot to remove this tag

File: data/pt_BR/system/dashboards/resources/gwt/schedulingdialogs/messages/mantleMessages_pt_BR.properties [hunk 2]
```diff
@@ -395,35 +395,35 @@ sunday=Sunday<TRANSLATE ME>
...
-lastFire=Last Run<TRANSLATE ME>
-nextFire=Next Run<TRANSLATE ME>
-nextStep=Next ><TRANSLATE ME>
-previousStep=< Back<TRANSLATE ME>
-user=Created By<TRANSLATE ME>
-state=Status<TRANSLATE ME>
+recurrence=Repetições
+lastFire=Execução Anterior
+nextFire=Proxima Execução
+nextStep=Pŕoximo ><TRANSLATE ME>
```
Comment (carlosrusso) [inline data/pt_BR/system/dashboards/resources/gwt/schedulingdialogs/messages/mantleMessages_pt_BR.properties]: oops

**Comment by carlosrusso** [comment]
Please push another commit that removes the <Translate me> tags and I'll merge the pull request

**Comment by marciojv** [comment]
Hello @carlosrusso  now is correct!

**Pull request merged**